pkg/virt-launcher/virtwrap/api/schema.go

```diff
@@ -273,9 +273,16 @@ type VCPUs struct {
 	VCPU []VCPUsVCPU `xml:"vcpu"`
 }
 
+type CPUModel struct {
```

Comment:
Do we need this struct? Or at least all the values. You are only setting the Value in the converting and not the rest